js/custom.js

```diff
@@ -44,6 +44,25 @@
       })
       e.preventDefault();
     });
+
+    $('#subscribe').click(function(e){
+      $.ajax({
+          url: "https://www.shop-mo.app/api/cosmotech/project",
```

Comment:
I haven't inplemented the news letter yet in the backend so this route is just for project reuests